charts/kubevirt/0.1.0/templates/kubevirt-cr-uninstaller.yaml

```
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: kubevirt
+  name: kubevirt-cr-uninstaller
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-weight": "1"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: kubevirt
+  name: kubevirt-cr-uninstaller
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-weight": "1"
+rules:
+  - apiGroups: [ "kubevirt.io" ]
+    resources: [ "kubevirts" ]
+    resourceNames:
+      - "kubevirt"
+    verbs: [ "delete" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: kubevirt
+  name: kubevirt-cr-uninstaller
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-weight": "2"
+subjects:
+  - kind: ServiceAccount
+    namespace: kubevirt
+    name: kubevirt-cr-uninstaller
+roleRef:
+  kind: Role
+  name: kubevirt-cr-uninstaller
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: kubevirt
+  name: kubevirt-cr-uninstaller
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-weight": "3"
+spec:
+  template:
+    metadata:
+      name: kubevirt-cr-uninstaller
+    spec:
+      serviceAccountName: kubevirt-cr-uninstaller
+      restartPolicy: OnFailure
+      containers:
+        - name: kubevirt-cr-uninstaller
+          image: {{ .Values.uninstaller.image }}
+          command:
+            - /bin/sh
+            - -c
+            - >
+              kubectl delete kubevirt kubevirt;
```

I think this should be namespaced
```suggestion
              kubectl delete -n kubevirt kubevirt kubevirt;
```

---

Currently executing the job from the `kubevirt` namespace which is why specifying the namespace is not required here.